deploy/kube-templates/heketi-deployment.yaml

```diff
@@ -1,4 +1,15 @@
 ---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: heketi-db-backup
+  labels:
+    glusterfs: heketi-db
+    heketi: db
```

Comment:
Could be helpful to search nothing in particular. Can remove not anything important, Do you want me to remove?